substrate/frame/ranked-collective/src/lib.rs

```diff
@@ -431,6 +432,14 @@ pub mod pallet {
 		/// in the poll.
 		type VoteWeight: Convert<Rank, Votes>;
 
+		/// The maximum number of members for a given rank in the collective.
+		///
+		/// The member at rank `x` contributes to the count at rank `x` and all ranks below it.
+		/// Therefore, the limit `m` at rank `x` sets the maximum total member count for rank `x`
+		/// and all ranks above.
+		/// The `None` indicates no member count limit for the given rank.
+		type MaxMemberCount: MaybeConvert<Rank, MemberIndex>;
```

Comment:
Do we have a default config for this pallet? Would be nice to set this to `()` to avoid the breaking change.

Reply:
No, it does not have. There is also a discussion to use default configs only for tests and maybe common configs for system parachains https://forum.polkadot.network/t/seeking-feedback-on-derive-impl-for-default-configurations-in-pallet-templates/7584

Reply:
also here https://github.com/paritytech/polkadot-sdk/pull/4689

Reply:
I think there should still be a trivial implementation of this to reduce copy&paste.  
Some struct like `UnlimitedMembersPerRank` or so that just always returns `None`.

Reply:
`MaybeConvert` has a tuple implementation, which always returns none

Reply:
Oh, so it is possible to just pass `()`? Then i misunderstood the discussion, good!